src/drivers/vmount/output_mavlink.cpp

```diff
@@ -113,9 +113,9 @@ int OutputMavlink::update(const ControlData *control_data)
 
 	// vmount spec has roll, pitch on channels 0, 1, respectively; MAVLink spec has roll, pitch on channels 1, 0, respectively
 	// vmount uses radians, MAVLink uses degrees
-	vehicle_command.param1 = _angle_outputs[1] * M_RAD_TO_DEG_F;
```

Comment:
Why did you remove the `* M_RAD_TO_DEG_F` factor?

Reply:
Oops, looks like I got a bit overenthusiastic when I changed the offset to be in radians, thanks.

Reply:
(resolved)